### PR TITLE
Add 'Content properties' section to Promo

### PR DESCRIPTION
**Pull request by simonsinclair**
A first stab at resolving #97. What do you think?

### CODE REVIEW

File: src/components/promos.md [hunk 1]
```diff
@@ -13,6 +13,19 @@ The **Promo** component is used to advertise BBC content. Principally it is a li
 
 The full range of supporting **Promo** content is outlined on the [GEL Promo page](https://www.bbc.co.uk/gel/guidelines/promos).
 
+## Content properties
+
+A **Promo** can be made up of different content, giving us the flexibility to communicate the right message. Below is a list of the content properties covered in this document.
+
+| Property | Description |
+| -------- | ----------- |
+| Headline | The target page's `<title>` and `<h1>`. |
```
[the remark this answers is not in the document]
Comment (simonsinclair) [reply src/components/promos.md]: Ah, yes. Updated.

File: src/components/promos.md [hunk 2]
```diff
@@ -13,6 +13,19 @@ The **Promo** component is used to advertise BBC content. Principally it is a li
...
+| Image | An image promoting the target content. |
+| Media indication | Information about the media being promoted. |
+| Description | A short description of the target content. |
+| Metadata | See the [Metadata strip](http://localhost:3000/gef-docs/components/metadata-strips) component. |
```
[the remark this answers is not in the document]
Comment (simonsinclair) [reply src/components/promos.md]: Schoolboy error. 😅  Fixed.

**Comment by simonsinclair** [comment]
@micmath

**Comment by micmath** [comment]
Thank you!

**Pull request merged**